picturegeneratorjs.js

```diff
@@ -0,0 +1,16 @@
+$(".leImage").on("click", function()
```

Comment:
You don't have any elements in your html file that contain a class called "letImage" so this can't be selected